Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.19
+
+WORKDIR /app
+
+COPY go.mod go.sum ./
+
+RUN go mod download
+
+COPY . .
+
+RUN CGO_ENABLED=0 GOOS=linux go build -o /cfnc
+
```

Comment:
We might need to introduce multi stage deployment for creating minimal docker image. I haven't tested it yet.
```Dockerfile
FROM golang:1.19 as builder

WORKDIR /app

COPY go.mod go.sum ./

RUN go mod download

COPY . .

RUN CGO_ENABLED=0 GOOS=linux go build -o cfnc

FROM scratch as final
COPY --from=builder /app/cfnc /cfnc

ENTRYPOINT [ "/cfnc" ]

```